<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CapacityConversionView</class>
 <widget class="QWidget" name="CapacityConversionView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Arial Unicode MS&quot;;</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>211</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="line1Edit">
          <property name="font">
           <font>
            <family>Arial Unicode MS</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox1">
          <item>
           <property name="text">
            <string>升</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>毫升</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>加仑</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>立方米</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>立方厘米</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="line2Edit">
          <property name="font">
           <font>
            <family>Arial Unicode MS</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox2">
          <item>
           <property name="text">
            <string>升</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>毫升</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>加仑</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>立方米</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>立方厘米</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>18</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>3</width>
        <height>6</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Unicode MS</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
